rep-2001.rst

```diff
@@ -773,6 +775,34 @@ Desktop Full
       packages: [perception, simulation, ros_ign_gazebo_demos]
 
 
+ROS Distribution Independent (September 2022 - Ongoing)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These variants are not released as ROS packages within each distribution as the others are.
+Instead they are made available as independent packages outside of any ROS distribution.
+
+Build Essential
+"""""""""""""""
+
+In addition to the explicitly listed packages this variant should include the base toolchain for any operating system (such as `build-essential`) on Debian as well as GNU Make and a C++ compiler if not already part of that toolchain.
```

Comment:
I don't think there is a way to include this in the extends/packages format below so I pulled the descriptive text from the implementation README in infra-variants. Extra eyes here are appreciated.

Reply:
I'm not looking at any of the implementation at the moment, but it seems like there should be a [`build-essential` entry in the `rosdep/base.yaml`](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml)

Reply:
> I'm not looking at any of the implementation at the moment, but it seems like there should be a [build-essential entry in the rosdep/base.yaml](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml)

I'm a bit conflicted about it at the moment. Since the infra-variants aren't ROS packages and don't use rosdep, it's not required for those packages to work and for Debian/Ubuntu specifically, I don't think anything is necessarily _supposed_ to depend on `build-essential` since it is meant to be assumed available when building any package and if a package has dependencies on specific contents of build-essential it should declare those instead.

The reason we're including it for ros-build-essential is that we intend a superset of it.

Since it's not required to have a rosdep key for it, it is not a blocker.